lesson_04/RockPaperScissors/main.py

```diff
@@ -0,0 +1,66 @@
+import random
+import sys
+
+
+def main():
+    r = "rock"
+    p = "paper"
+    s = "scissors"
+    rps = [r, p, s]
+
+    print("rock paper scissors!")
+    print("choose your weapon: Rock, Paper, Scissors or Q to quit:")
+    user_inp = input()
```

Comment:
Please use full words in variable name.

a.k.a. rename user_inp to user_input, pc_inp to pc_input, r to rock, etc.
You can use the Refactor->Rename thingy to do this quickly and without errors